### PR TITLE
Remove config.js in favor of .env

### PR DESCRIPTION
**Pull request by phyrog**
Makes it easier to inject build time options like the backend host by setting the `REACT_APP_BACKEND_HOST` environment variable. See ontohub/deployment#9.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1 +1,4 @@
 PORT=3001
+
+REACT_APP_BACKEND_HOST="http://localhost:3000"
+REACT_APP_BACKEND_VERSION="> 0.0.0-90"
```
Comment (eugenk) [inline .env]: I don't think that the backend version requirement should be part of the `.env` file. It is a requirement that *the code* makes, not one that *the server* makes. Maybe we should keep a `config.js` for things like this and have an additional `.env` file for things like the backend host address.

**Comment by eugenk** [comment]
Since you are fixing the frontend-backend communication here, can you please add a request header `Accept: application/json` to every request that is made to the backend?

File: .env (hunk 1 above)
Comment (phyrog) [inline .env]: I think a good solution would be to specify the backend version in the `package.json` file (custom fields are allowed), but importing the file (and all other files outside of `src`) is currently not allowed. There is [a PR](https://github.com/facebookincubator/create-react-app/pull/2468) for that however that seems to be near completion.

Comment (phyrog) [inline .env]: As a work-around for now, I can symlink `package.json` to `src/config.json`. Any downsides to symlinks in git?

**Pull request merged**